### PR TITLE
Chore: e2e tests on mcq component (Issue/218)

### PR DESCRIPTION
**Pull request by lemmyadams**
Addresses #218

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by lemmyadams** [comment]
Relies on https://github.com/adaptlearning/adapt_framework/pull/3533

**Review by ethan-lp — APPROVED** [review]

**Comment by oliverfoster** [comment]
Brackets, line endings and odd whitespace and compiled html properties.

File: test/e2e/mcq.cy.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+describe('Multiple Choice Question', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the mcq component', function () {
+    const mcqComponents = this.data.components.filter((component) => component._component === 'mcq')
+    mcqComponents.forEach((mcqComponent) => {
```
Comment (joe-allen-89) [inline test/e2e/mcq.cy.js:9]: We can remove the additional brackets from the arrow functions. 

```suggestion
    const mcqComponents = this.data.components.filter(component => component._component === 'mcq')
    mcqComponents.forEach(mcqComponent => {
```

File: test/e2e/mcq.cy.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      cy.visit(`/#/preview/${mcqComponent._id}`);
+      const bodyWithoutHtml = cy.helpers.stripHtml(mcqComponent.body)
+      cy.testContainsOrNotExists('.mcq__body', bodyWithoutHtml)
+
+      cy.testContainsOrNotExists('.mcq__title', mcqComponent.displayTitle)
+      cy.testContainsOrNotExists('.mcq__instruction', mcqComponent.instruction)
+      cy.get('.mcq-item').should('have.length', mcqComponent._items.length)
+      mcqComponent._items.forEach((item) => {
```
Comment (joe-allen-89) [inline test/e2e/mcq.cy.js:17]:
```suggestion
      mcqComponent._items.forEach(item => {
```

**Review by joe-allen-89 — CHANGES_REQUESTED** [review]

File: test/e2e/mcq.cy.js [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-      const bodyWithoutHtml = cy.helpers.stripHtml(mcqComponent.body)
-      cy.testContainsOrNotExists('.mcq__body', bodyWithoutHtml)
+      const stripHtml = cy.helpers.stripHtml;
+      cy.testContainsOrNotExists('.mcq__body', stripHtml(mcqComponent.body));
+      cy.testContainsOrNotExists('.mcq__title', stripHtml(mcqComponent.displayTitle));
+      cy.testContainsOrNotExists('.mcq__instruction', stripHtml(mcqComponent.instruction));
+
+      cy.get('.mcq-item').should('have.length', mcqComponent._items.length);
+      mcqComponent._items.forEach(item => {
+        cy.testContainsOrNotExists('.mcq-item__text', item.text);
```
Comment (oliverfoster) [inline test/e2e/mcq.cy.js:18]:
```suggestion
        cy.testContainsOrNotExists('.mcq-item__text', stripHtml(item.text));
```

Also compiled html 
https://github.com/adaptlearning/adapt-contrib-mcq/blob/f585154e6778e065855bea97365c1792f0b8af2e/templates/mcq.jsx#L109

**Pull request merged**